{
	"id": "172834",
	"key": "TIMOB-26982",
	"fields": {
		"issuetype": {
			"id": "1",
			"description": "A problem which impairs or prevents the functions of the product.",
			"name": "Bug",
			"subtask": false
		},
		"project": {
			"id": "10153",
			"key": "TIMOB",
			"name": "Titanium SDK/CLI",
			"projectCategory": {
				"id": "10100",
				"description": "Titanium and related SDKs used in application development",
				"name": "Client"
			}
		},
		"fixVersions": [],
		"resolution": {
			"id": "3",
			"description": "The problem is a duplicate of an existing issue.",
			"name": "Duplicate"
		},
		"resolutiondate": "2019-04-12T01:09:26.000+0000",
		"created": "2019-01-09T11:43:23.000+0000",
		"priority": {
			"name": "Critical",
			"id": "1"
		},
		"labels": [
			"android"
		],
		"versions": [],
		"issuelinks": [
			{
				"id": "57495",
				"type": {
					"id": "10002",
					"name": "Duplicate",
					"inward": "is duplicated by",
					"outward": "duplicates"
				},
				"outwardIssue": {
					"id": "130892",
					"key": "TIMOB-17089",
					"fields": {
						"summary": "Android: Activity/Fragment restore can sometimes crash app due to view ID collision",
						"status": {
							"description": "The issue is considered finished, the resolution is correct. Issues which are closed can be reopened.",
							"name": "Closed",
							"id": "6",
							"statusCategory": {
								"id": 3,
								"key": "done",
								"colorName": "green",
								"name": "Done"
							}
						},
						"priority": {
							"name": "Critical",
							"id": "1"
						},
						"issuetype": {
							"id": "1",
							"description": "A problem which impairs or prevents the functions of the product.",
							"name": "Bug",
							"subtask": false
						}
					}
				}
			}
		],
		"assignee": {
			"name": "jquick",
			"key": "jquick",
			"displayName": "Joshua Quick",
			"active": true,
			"timeZone": "America/Los_Angeles"
		},
		"updated": "2019-04-12T01:09:26.000+0000",
		"status": {
			"description": "The issue is considered finished, the resolution is correct. Issues which are closed can be reopened.",
			"name": "Closed",
			"id": "6",
			"statusCategory": {
				"id": 3,
				"key": "done",
				"colorName": "green",
				"name": "Done"
			}
		},
		"components": [
			{
				"id": "10202",
				"name": "Android",
				"description": "Android Platform"
			}
		],
		"attachment": [],
		"flagged": false,
		"summary": "Android: java.lang.ClassCastException: ti.modules.titanium.ui.widget.TiUILabel$1 cannot be cast to android.view.ViewGroup;",
		"creator": {
			"name": "felipemathies",
			"key": "felipemathies",
			"displayName": "Felipe Mathies",
			"active": true,
			"timeZone": "America/Sao_Paulo"
		},
		"subtasks": [],
		"reporter": {
			"name": "felipemathies",
			"key": "felipemathies",
			"displayName": "Felipe Mathies",
			"active": true,
			"timeZone": "America/Sao_Paulo"
		},
		"environment": "Titanium 7.5.1.v20181218070137",
		"comment": {
			"comments": [
				{
					"id": "445138",
					"author": {
						"name": "sdarda",
						"key": "sdarda",
						"displayName": "Sharif AbuDarda",
						"active": false,
						"timeZone": "Asia/Dhaka"
					},
					"body": "Hello, Was this an app built on the previous SDK, CLI and now having the issue with the latest SDK? Please share your environment details with \"appc info\" command log. Thanks.",
					"updateAuthor": {
						"name": "sdarda",
						"key": "sdarda",
						"displayName": "Sharif AbuDarda",
						"active": false,
						"timeZone": "Asia/Dhaka"
					},
					"created": "2019-01-09T17:05:42.000+0000",
					"updated": "2019-01-09T17:05:42.000+0000"
				},
				{
					"id": "445202",
					"author": {
						"name": "rspedrini",
						"key": "rspedrini",
						"displayName": "Rafael Pedrini",
						"active": true,
						"timeZone": "America/Sao_Paulo"
					},
					"body": "Hello Sharif, \r\n\r\nI work with Felipe, our App was created using earlier SDK versions, some years ago, then since last month we are updating to most recent SDK version, the last SDK we used was 7.1.1. Officially now we are using SDK 7.5.0.GA, but we already test some nightly builds of 7.5.1, without success to solve this problem.\r\n\r\nappc info result:\r\n{noformat}\r\nOperating System\r\n  Name                        = Mac OS X\r\n  Version                     = 10.13.6\r\n  Architecture                = 64bit\r\n  # CPUs                      = 4\r\n  Memory                      = 8.0GB\r\n\r\nNode.js\r\n  Node.js Version             = 8.11.3\r\n  npm Version                 = 6.4.0\r\n\r\nAppcelerator CLI\r\n  Installer                   = 4.2.13\r\n  Core Package                = 7.0.6\r\n\r\nTitanium CLI\r\n  CLI Version                 = 5.1.1\r\n  node-appc Version           = 0.2.44\r\n{noformat}\r\n",
					"updateAuthor": {
						"name": "rspedrini",
						"key": "rspedrini",
						"displayName": "Rafael Pedrini",
						"active": true,
						"timeZone": "America/Sao_Paulo"
					},
					"created": "2019-01-11T03:28:25.000+0000",
					"updated": "2019-01-11T03:37:55.000+0000"
				},
				{
					"id": "445226",
					"author": {
						"name": "sdarda",
						"key": "sdarda",
						"displayName": "Sharif AbuDarda",
						"active": false,
						"timeZone": "Asia/Dhaka"
					},
					"body": "Hello, Try with the latest CLI 7.0.9. Run \"appc setup\" to install the latest CLI. Thanks.",
					"updateAuthor": {
						"name": "sdarda",
						"key": "sdarda",
						"displayName": "Sharif AbuDarda",
						"active": false,
						"timeZone": "Asia/Dhaka"
					},
					"created": "2019-01-11T17:37:44.000+0000",
					"updated": "2019-01-11T17:37:44.000+0000"
				},
				{
					"id": "445245",
					"author": {
						"name": "rspedrini",
						"key": "rspedrini",
						"displayName": "Rafael Pedrini",
						"active": true,
						"timeZone": "America/Sao_Paulo"
					},
					"updateAuthor": {
						"name": "rspedrini",
						"key": "rspedrini",
						"displayName": "Rafael Pedrini",
						"active": true,
						"timeZone": "America/Sao_Paulo"
					},
					"created": "2019-01-12T20:54:42.000+0000",
					"updated": "2019-01-12T20:54:42.000+0000"
				},
				{
					"id": "445248",
					"author": {
						"name": "michael",
						"key": "michael",
						"displayName": "Michael Gangolf",
						"active": true,
						"timeZone": "Europe/Berlin"
					},
					"body": "Yes, very hard to reproduce but it happens even in the latest SDK. Have that in some apps, too. But can't reproduce it in examples.",
					"updateAuthor": {
						"name": "michael",
						"key": "michael",
						"displayName": "Michael Gangolf",
						"active": true,
						"timeZone": "Europe/Berlin"
					},
					"created": "2019-01-12T22:05:43.000+0000",
					"updated": "2019-01-12T22:05:43.000+0000"
				},
				{
					"id": "445254",
					"author": {
						"name": "michael",
						"key": "michael",
						"displayName": "Michael Gangolf",
						"active": true,
						"timeZone": "Europe/Berlin"
					},
					"body": "[~sdarda] btw: I gave Yordan a full app of mine that suffers from this bug, too.\r\n{code}\r\n[ERROR] TiExceptionHandler: (main) [156856,156856] Unable to start activity ComponentInfo{com.miga.app/org.appcelerator.titanium.TiActivity}: java.lang.ClassCastException: ti.modules.titanium.ui.widget.TiUILabel$1 cannot be cast to android.view.ViewGroup\r\n[ERROR] TiExceptionHandler:\r\n[ERROR] TiExceptionHandler:     android.support.v4.app.FragmentTransition.configureTransitionsOrdered(FragmentTransition.java:310)\r\n[ERROR] TiExceptionHandler:     android.support.v4.app.FragmentTransition.startTransitions(FragmentTransition.java:134)\r\n[ERROR] TiExceptionHandler:     android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2380)\r\n[ERROR] TiExceptionHandler:     android.support.v4.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2338)\r\n[ERROR] TiExceptionHandler:     android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2245)\r\n[ERROR] TiExceptionHandler:     android.support.v4.app.FragmentManagerImpl.dispatchStateChange(FragmentManager.java:3248)\r\n[ERROR] TiExceptionHandler:     android.support.v4.app.FragmentManagerImpl.dispatchActivityCreated(FragmentManager.java:3200)\r\n[ERROR] TiExceptionHandler:     android.support.v4.app.FragmentController.dispatchActivityCreated(FragmentController.java:195)\r\n[ERROR] TiExceptionHandler:     android.support.v4.app.FragmentActivity.onStart(FragmentActivity.java:597)\r\n[ERROR] TiExceptionHandler:     android.support.v7.app.AppCompatActivity.onStart(AppCompatActivity.java:177)\r\n[ERROR] TiExceptionHandler:     org.appcelerator.titanium.TiBaseActivity.onStart(TiBaseActivity.java:1504)\r\n[ERROR] TiExceptionHandler:     android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1299)\r\n[ERROR] TiExceptionHandler:     android.app.Activity.performStart(Activity.java:6690)\r\n[ERROR] TiExceptionHandler:     android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2688)\r\n[ERROR] TiExceptionHandler:     android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2786)\r\n[ERROR] TiExceptionHandler:     android.app.ActivityThread.-wrap12(ActivityThread.java)\r\n[ERROR] TiExceptionHandler:     android.app.ActivityThread$H.handleMessage(ActivityThread.java:1501)\r\n[ERROR] TiExceptionHandler:     android.os.Handler.dispatchMessage(Handler.java:102)\r\n[ERROR] TiExceptionHandler:     android.os.Looper.loop(Looper.java:173)\r\n[ERROR] TiExceptionHandler:     android.app.ActivityThread.main(ActivityThread.java:6459)\r\n[ERROR] TiExceptionHandler:     java.lang.reflect.Method.invoke(Native Method)\r\n[ERROR] TiExceptionHandler:     com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:938)\r\n[ERROR] TiExceptionHandler:     com.android.internal.os.ZygoteInit.main(ZygoteInit.java:828)\r\n{code}\r\n\r\nSDK 7.5.1.v20181210141225\r\n\r\nSo he might be able to run some internal tests",
					"updateAuthor": {
						"name": "michael",
						"key": "michael",
						"displayName": "Michael Gangolf",
						"active": true,
						"timeZone": "Europe/Berlin"
					},
					"created": "2019-01-13T12:37:18.000+0000",
					"updated": "2019-01-13T12:38:01.000+0000"
				},
				{
					"id": "445258",
					"author": {
						"name": "rspedrini",
						"key": "rspedrini",
						"displayName": "Rafael Pedrini",
						"active": true,
						"timeZone": "America/Sao_Paulo"
					},
					"body": "Michael, I found another App (that was not made with Titanium) with this crash: https://github.com/wordpress-mobile/WordPress-Android/issues/7973\r\nThe stacktrace is almost the same.\r\n\r\nI already removed all my labels from screen to test (desperate test), crash still happens...\r\n\r\nEverybody that uses SDK 7.5.0.GA or later are experiencing this problem?",
					"updateAuthor": {
						"name": "rspedrini",
						"key": "rspedrini",
						"displayName": "Rafael Pedrini",
						"active": true,
						"timeZone": "America/Sao_Paulo"
					},
					"created": "2019-01-13T15:27:56.000+0000",
					"updated": "2019-01-13T15:28:58.000+0000"
				},
				{
					"id": "445259",
					"author": {
						"name": "michael",
						"key": "michael",
						"displayName": "Michael Gangolf",
						"active": true,
						"timeZone": "Europe/Berlin"
					},
					"body": "not everybody has that error and it was in earlier SDKs too (see the old ticket mentioned in the description).\r\nThe last line that is mentioned in the error is pointing to:\r\nhttps://android.googlesource.com/platform/frameworks/support/+/oreo-release/fragment/java/android/support/v4/app/FragmentTransition.java#293\r\nThe interesting part for me is: why does it work most of the time (is it not trying to run the transition on the elements or do they work?) and what is causing the problem that it can't convert them.",
					"updateAuthor": {
						"name": "michael",
						"key": "michael",
						"displayName": "Michael Gangolf",
						"active": true,
						"timeZone": "Europe/Berlin"
					},
					"created": "2019-01-13T15:58:57.000+0000",
					"updated": "2019-01-13T15:58:57.000+0000"
				},
				{
					"id": "445260",
					"author": {
						"name": "rspedrini",
						"key": "rspedrini",
						"displayName": "Rafael Pedrini",
						"active": true,
						"timeZone": "America/Sao_Paulo"
					},
					"body": "Is there any way that we can recompile this source to debug and use it? Maybe a try catch to show us the name of component that is causing the exception...\r\n\r\nOur App have at least 20 different screens, this crash occur only in 4 or 5 screens and its completely random, I already built a version then I repeated the same operation more than 300 times and had no crashs, then 10 minutes later the crash occured in third action.",
					"updateAuthor": {
						"name": "rspedrini",
						"key": "rspedrini",
						"displayName": "Rafael Pedrini",
						"active": true,
						"timeZone": "America/Sao_Paulo"
					},
					"created": "2019-01-13T17:58:01.000+0000",
					"updated": "2019-01-13T17:59:04.000+0000"
				},
				{
					"id": "445261",
					"author": {
						"name": "rspedrini",
						"key": "rspedrini",
						"displayName": "Rafael Pedrini",
						"active": true,
						"timeZone": "America/Sao_Paulo"
					},
					"body": "Michael, are you using Ti.Map in your project? I think that Ti.Map can be related with this crashs in my scenario...",
					"updateAuthor": {
						"name": "rspedrini",
						"key": "rspedrini",
						"displayName": "Rafael Pedrini",
						"active": true,
						"timeZone": "America/Sao_Paulo"
					},
					"created": "2019-01-14T02:22:08.000+0000",
					"updated": "2019-01-14T02:22:08.000+0000"
				},
				{
					"id": "445266",
					"author": {
						"name": "michael",
						"key": "michael",
						"displayName": "Michael Gangolf",
						"active": true,
						"timeZone": "Europe/Berlin"
					},
					"body": "Yes, [~ybanev] was investigating in that direction too. I had a Label placed on the Map, removing that was promising at first but after a few tests the error still appeared. ",
					"updateAuthor": {
						"name": "michael",
						"key": "michael",
						"displayName": "Michael Gangolf",
						"active": true,
						"timeZone": "Europe/Berlin"
					},
					"created": "2019-01-14T07:17:29.000+0000",
					"updated": "2019-01-14T07:17:29.000+0000"
				},
				{
					"id": "445789",
					"author": {
						"name": "sdarda",
						"key": "sdarda",
						"displayName": "Sharif AbuDarda",
						"active": false,
						"timeZone": "Asia/Dhaka"
					},
					"body": "Hello [~michael], Can you please follow up here? Whats the current status of your issue?",
					"updateAuthor": {
						"name": "sdarda",
						"key": "sdarda",
						"displayName": "Sharif AbuDarda",
						"active": false,
						"timeZone": "Asia/Dhaka"
					},
					"created": "2019-01-31T17:49:21.000+0000",
					"updated": "2019-01-31T17:49:21.000+0000"
				},
				{
					"id": "445799",
					"author": {
						"name": "michael",
						"key": "michael",
						"displayName": "Michael Gangolf",
						"active": true,
						"timeZone": "Europe/Berlin"
					},
					"body": "I don't have any updates regarding the investigations on your side. The error still appears in my app (which Yordan has a copy of) and I'm not sure what he found out. So at the moment I'm waiting for his feedback :) ",
					"updateAuthor": {
						"name": "michael",
						"key": "michael",
						"displayName": "Michael Gangolf",
						"active": true,
						"timeZone": "Europe/Berlin"
					},
					"created": "2019-02-01T15:07:39.000+0000",
					"updated": "2019-02-01T15:07:39.000+0000"
				},
				{
					"id": "447631",
					"author": {
						"name": "jquick",
						"key": "jquick",
						"displayName": "Joshua Quick",
						"active": true,
						"timeZone": "America/Los_Angeles"
					},
					"body": "I was able to reproduce this issue with [~michael]'s project.\r\nThe fix for [TIMOB-17089] resolves this issue. Closing as duplicate.",
					"updateAuthor": {
						"name": "jquick",
						"key": "jquick",
						"displayName": "Joshua Quick",
						"active": true,
						"timeZone": "America/Los_Angeles"
					},
					"created": "2019-04-12T01:09:07.000+0000",
					"updated": "2019-04-12T01:09:07.000+0000"
				}
			],
			"maxResults": 14,
			"total": 14,
			"startAt": 0
		}
	}
}